packages/core/src/utils/Target.ts

```diff
@@ -0,0 +1,19 @@
+import { DebugEvent } from '../types';
```

Comment:
Personally I'd move this to a global instead of saving it on the client, this supports debug warnings that could go across multiple roots which would be awesome.